rosbag2_transport/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ endif()
 # Windows supplies macros for min and max by default. We should only use min and max from stl
 if(WIN32)
   add_definitions(-DNOMINMAX)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj")
```

Comment:
Is there a specific reason not to use `add_compile_options` as was done above?

Reply:
I'm not sure.  I copied it from where we do similar things in `rviz_default_plugins` and `test_communication`, which is why it is like this.  I can try it out with `add_compile_options`, if you think that is valuable.

Reply:
I don't really care, was just curious.